mautic/mautic
# Fix `mautic:disable-tracking `

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | "features" for all features, enhancements and bug fixes (until 3.3.0 is released) 
| Bug fix?                               | yes/no
| New feature?                           | yes/no
| Deprecations?                          | yes/no
| BC breaks?                             | yes/no
| Automated tests included?              | yes/no
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #7173... 




#### Description:



#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Try to add a link that should not be tracked
3. The legecny builder should not trim the attribute `mautic:disable-tracking`
